ticket_call_details.view.lkml

```diff
@@ -0,0 +1,180 @@
+view: ticket_call_details {
+  derived_table: {
+    sql: SELECT
+        audits.id,
+        ticket_id,
+        events.data__call_duration AS call_duration,
+        events.data__answered_by_name AS agent_on_call,
```

Comment:
Yes, I switched this to agent_id and instead of bringing in agent_name to this view, will join the users table.  It's now a similar set up to the satisfaction_rating view

Reply:
<img width="221" alt="screen shot 2018-11-21 at 11 08 59 am" src="https://user-images.githubusercontent.com/38991765/48863359-ef858500-ed7d-11e8-9a57-5cf54ea7421a.png">